lib/recurly/purchase.php

```diff
@@ -54,11 +54,43 @@ public static function preview($purchase, $client = null) {
    * @param Recurly_Client $client Optional client for the request, useful for mocking the client
    * @return object Recurly_InvoiceCollection
    * @throws Recurly_Error
+   *
+   * Note: To use this endpoint, you may have to contact Recurly support to have it enabled on your subdomain.
    */
   public static function authorize($purchase, $client = null) {
     return Recurly_Base::_post('/purchases/authorize', $purchase->xml(), $client);
   }
 
+  /**
+   *
+   * Capture an open Authorization request
+   *
+   * @param $purchase Recurly_Purchase Our purchase data.
+   * @param Recurly_Client $client Optional client for the request, useful for mocking the client
+   * @return object Recurly_InvoiceCollection
+   * @throws Recurly_Error
+   *
+   * Note: To use this endpoint, you may have to contact Recurly support to have it enabled on your subdomain.
+   */
+  public static function capture($transactionUUID, $client = null) {
+    return Recurly_Base::_post("/purchases/transaction-uuid-$transactionUUID/capture", null, $client);
+  }
+
+  /**
+   *
+   * Cancel an open Authorization request
+   *
+   * @param $purchase Recurly_Purchase Our purchase data.
+   * @param Recurly_Client $client Optional client for the request, useful for mocking the client
+   * @return object Recurly_InvoiceCollection
+   * @throws Recurly_Error
+   *
+   * Note: To use this endpoint, you may have to contact Recurly support to have it enabled on your subdomain.
+   */
+  public static function cancel($transactionUUID, $client = null) {
+      return Recurly_Base::_post("/purchases/transaction-uuid-$transactionUUID/cancel", null, $client);
```

Comment:
Nit: Is there a reason this return statement is indented with 6 spaces instead of 4 like the rest of the file?